.changeset/pre.json

```
@@ -1,19 +1,21 @@
 {
   "mode": "pre",
-  "tag": "alpha",
+  "tag": "beta",
   "initialVersions": {
-    "@hyperlane-xyz/core": "4.0.0-alpha.0",
-    "@hyperlane-xyz/ccip-server": "4.0.0-alpha.0",
-    "@hyperlane-xyz/cli": "4.0.0-alpha.0",
-    "@hyperlane-xyz/helloworld": "4.0.0-alpha.0",
-    "@hyperlane-xyz/infra": "4.0.0-alpha.0",
-    "@hyperlane-xyz/sdk": "4.0.0-alpha.0",
-    "@hyperlane-xyz/utils": "4.0.0-alpha.0"
+    "@hyperlane-xyz/core": "4.0.0-alpha.2",
+    "@hyperlane-xyz/ccip-server": "4.0.0-alpha.2",
+    "@hyperlane-xyz/cli": "4.0.0-alpha.2",
+    "@hyperlane-xyz/helloworld": "4.0.0-alpha.2",
+    "@hyperlane-xyz/infra": "4.0.0-alpha.2",
+    "@hyperlane-xyz/sdk": "4.0.0-alpha.2",
+    "@hyperlane-xyz/utils": "4.0.0-alpha.2"
```

should this be updated too? ```suggestion
    "@hyperlane-xyz/core": "4.0.0-beta",
    "@hyperlane-xyz/ccip-server": "4.0.0-beta",
    "@hyperlane-xyz/cli": "4.0.0-beta",
    "@hyperlane-xyz/helloworld": "4.0.0-beta",
    "@hyperlane-xyz/infra": "4.0.0-beta",
    "@hyperlane-xyz/sdk": "4.0.0-beta",
    "@hyperlane-xyz/utils": "4.0.0-beta"
```

---

this file is generated and I think this is actually not intended to be changed there
this is like "what version were you in before you entered prerelease mode" I believe